cocotb/config.py

```diff
@@ -28,13 +28,33 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
+"""Module for querying the cocotb configuration
+
+This module provides information in module global variables and through a
+``main()`` function that is used in the cocotb-config script.
+
+Global variables:
+    version: str, the cocotb version
+    prefix_dir: str, path where cocotb module is installed
```

Comment:
I'm not actually sure we want to expose either of these here. The version should just be at `cocotb.__version__`, and the `prefix_dir` doesn't strike me as useful - the user can always just use `cocotb.__file__` themselves